Ghidra/Processors/PowerPC/data/languages/ppc_instructions.sinc

```diff
@@ -1930,8 +1930,9 @@
 	zero:4=0;
 	fD=fC;
 	cr1flags();
```

Comment:
should this line be removed then?

Reply:
I believe that line should stay, as the `goto inst_next` will skip over the final two lines and thus cr1flags() would not be called if `fA >= 0` (unless I misunderstood the behaviour of `goto inst_next`?). Since `cr1flags` has no side-effects, it doesn't matter that it might be called twice.

Reply:
oops, didnt click that its writing to fD twice, ignore this